### PR TITLE
Restore the web component studio

### PR DESCRIPTION
**Pull request by fabienrohrer**
Ditto

### CODE REVIEW

**Comment by fabienrohrer** [comment]
Simply restore the robot component from the Webots private repo without modification. I checked it's working as before.

**Comment by fabienrohrer** [comment]
test me

**Comment by fabienrohrer** [comment]
👍

**Comment by omichel** [comment]
I have some doubt regarding the terminology. Component doesn't mean very much...

**Comment by fabienrohrer** [comment]
In modern web terminology "component" is used a lot precisely to determine this kind of embedded app, for example in ReactJS: https://reactjs.org/docs/react-component.html

**Comment by DavidMansolino** [comment]
> In modern web terminology "component" is used a lot precisely to determine this kind of embedded app, for example in ReactJS: https://reactjs.org/docs/react-component.html

:+1: 
I remember we discussed this a few months ago and you and Tom had very convincing arguments about it.

**Comment by omichel** [comment]
Does it refer to a web component as defined for example here: https://www.webcomponents.org?

**Comment by fabienrohrer** [comment]
Ok, let's rename this to `web_component_studio`

**Comment by fabienrohrer** [comment]
test me

**Comment by fabienrohrer** [comment]
👍

**Comment by omichel** [comment]
👀

File: scripts/web_component_studio/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Web Component Studio
+
+The Web Component Studio is a script to create the Web components displayed in the online documentation.
```
Comment (omichel) [inline scripts/web_component_studio/README.md]:
```suggestion
The Web Component Studio is a script used to create the 3D web components displayed in the online documentation.
```

File: scripts/web_component_studio/README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+A Web component is a view of a Webots model, typically a robot.
```
Comment (omichel) [inline scripts/web_component_studio/README.md]:
```suggestion
A web component is an interactive 3D view of a Webots model, typically a robot.
```

File: scripts/web_component_studio/README.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+In addition of a 3D view of the Webots model, the Web component displays the list of the robot devices.
```
Comment (omichel) [inline scripts/web_component_studio/README.md]:
```suggestion
If the model is a robot, the web component also displays the list of devices of that robot, in addition to the 3D view of the robot.
```

File: scripts/web_component_studio/README.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+Motors can be moved using sliders.
+The device origin location are displayed when the mouse is over an item of the device list.
+
+To illustrate this, a Web component showing the Aibo ERS7 model can be shown on the top of the following page: https://cyberbotics.com/doc/guide/aibo-ers7
```
Comment (omichel) [inline scripts/web_component_studio/README.md]:
```suggestion
An example of a web component showing the Aibo ERS7 robot is displayed on the top of the following page: https://cyberbotics.com/doc/guide/aibo-ers7
```

**Comment by fabienrohrer** [comment]
test me

**Comment by fabienrohrer** [comment]
Thank you, I applied your suggestions as-is.
I also remarked that the `scripts` directory was not in the license test, and fixed this accordingly.

**Comment by fabienrohrer** [comment]
👍

**Pull request merged**